lib/matplotlib/_docstring.pyi

```diff
@@ -10,7 +10,7 @@ class Substitution:
     @overload
     def __init__(self, **kwargs: str): ...
     def __call__(self, func: _T) -> _T: ...
-    def update(self, *args, **kwargs): ...  # type: ignore[no-untyped-def]
+    def update(self, *args, **kwargs:Any): ...  # type: ignore[no-untyped-def]
```

Comment:
```suggestion
    def update(self, *args, **kwargs:Any): ...
```

I _think_ this type ignore can be removed at this point

Reply:
Thanks for the feedback and the tip about the PR. I will implement your suggestions regarding the style and *args, and also rerun tools/boilerplate.py, and update the Pull Request.